CMakeLists.txt

```diff
@@ -365,6 +370,12 @@ if(LIBVNCSERVER_WITH_WEBSOCKETS)
   )
 endif(LIBVNCSERVER_WITH_WEBSOCKETS)
 
+if(LIBVNCSERVER_WITH_WEBSOCKETS AND NOT HAVE_B64_IN_LIBC AND NOT HAVE_B64_IN_LIBRESOLV)
```

Comment:
Are you saying that you would like to always use local compiled version of `b64_ntop` and `b64_pton`, even if they are available in libc or libresolv?